omnigibson/systems/system_base.py

```diff
@@ -119,9 +220,206 @@ def get_active_systems(cls):
         """
         return {system.name: system for system in SYSTEM_REGISTRY.objects if issubclass(system, cls)}
 
+    @classmethod
+    def set_scale_limits(cls, minimum=None, maximum=None):
+        """
+        Set the min and / or max scaling limits that will be uniformly sampled from when generating new particles.
+        Note: These scales dictate the scale set for particles in their parents' frame, NOT the global frame!
```

Comment:
Let's remove this function, and add some comments to the min_scale/max_scale variables. 

I think these are actually the desired scale for the particle templates in the global frame, not the parent's object frame.